utils/zerovec/src/zerovec/mod.rs

```diff
@@ -346,38 +346,12 @@ where
     ///
     /// # Examples
     ///
-    /// Convert a borrowed `ZeroVec`:
-    ///
-    /// ```
-    /// use zerovec::ZeroVec;
-    ///
-    /// let bytes: &[u8] = &[0x7F, 0xF3, 0x01, 0x00, 0x49, 0xF6, 0x01, 0x00];
-    /// let zv_char: ZeroVec<char> = ZeroVec::parse_byte_slice(bytes).expect("valid code points");
-    /// let zv_u32: ZeroVec<u32> = zv_char.try_into_converted().expect("infallible conversion");
-    ///
-    /// assert!(matches!(zv_u32, ZeroVec::Borrowed(_)));
```

Comment:
hmm, i wonder if we can make this example use something else, like u32 vs i32

Reply:
How about keeping the same example but making the reference zerovec be `ZeroVec<[u8; 3]>` instead of `ZeroVec<u32>`

Reply:
CharULE requires three bytes, that's what matters here.

Reply:
Probably a separate bug to add tests to this doc? Can just remove them here for now, since char and integers are very different now, since we are throwing 1 byte out

Reply:
I don't want to leave the function untested though. I think an example working with u32 and i32 would make sense.

Reply:
I prefer using a `char` example because it illustrates the putpose of this function: to map from one zerovec to another zerovec with additional validation.